stdlib/http/src/main/ballerina/src/http/websocket/websocket_client.bal

```diff
@@ -60,47 +59,51 @@ public type WebSocketClient client object {
     #
     # + data - Binary data to be sent
     # + finalFrame - Set to `true` if this is a final frame of a (long) message
-    # + return - `error` if an error occurs when sending
+    # + return - Returns a `WebSocketError` if an error occurs while sending the binary message to the server.
+    #   The message will be lost if an error occurs.
     public remote function pushBinary(byte[] data, public boolean finalFrame = true) returns error? {
         return self.conn.pushBinary(data, finalFrame);
     }
 
-    # Ping the connection.
+    # Pings the connection.
     #
     # + data - Binary data to be sent.
-    # + return - `error` if an error occurs when sending
+    # + return - Returns a`WebSocketError` if an error occurs while sending the ping frame to the server.
+    #   The frame will be lost if an error occurs.
     public remote function ping(byte[] data) returns WebSocketError? {
         return self.conn.ping(data);
     }
 
-    # Send pong message to the connection.
+    # Sends a pong message to the connection.
     #
     # + data - Binary data to be sent
-    # + return - `error` if an error occurs when sending
+    # + return - Returns a `WebSocketError` if an error occurs while sending the pong frame to the server.
+    #   The frame will be lost if an error occurs.
     public remote function pong(byte[] data) returns WebSocketError? {
         return self.conn.pong(data);
     }
 
-    # Close the connection.
+    # Closes the connection.
     #
-    # + statusCode - Status code for closing the connection
+    # + statusCode - Status code for closing the connection.
```

Comment:
Ballerina documentation best practice is not to have fullstop at the end of field description. (It should be there in the record description) Check all the places.